lnrpc/rpc.proto

```diff
@@ -1496,14 +1570,10 @@ message NetworkInfo {
     uint32 num_nodes = 4 [json_name = "num_nodes"];
     uint32 num_channels = 5 [json_name = "num_channels"];
 
-    int64 total_network_capacity = 6 [json_name = "total_network_capacity"];
-
-    double avg_channel_size = 7 [json_name = "avg_channel_size"];
-    int64 min_channel_size = 8 [json_name = "min_channel_size"];
-    int64 max_channel_size = 9 [json_name = "max_channel_size"];
-
     // TODO(roasbeef): fee rate info, expiry
     //  * also additional RPC for tracking fee info once in
+
+    repeated ChainNetworkInfo chains = 10 [json_name = "chains"];
```

Comment:
Change index from 10 to 6.

Reply:
it is 10 on purpose. see earlier comment